crates/cli/src/scan.rs

```diff
@@ -34,10 +34,10 @@ pub struct ScanArg {
 
   /// Scan the codebase with specified subset of RULE_IDs only.
   ///
-  /// This flags conflicts with --rule. It is useful to only specific rules from a large set of
-  /// rule definitions within a project.
-  #[clap(short = 'R', long, conflicts_with = "rule", value_name = "RULE_ID")]
-  rule_id: Vec<String>,
+  /// This flags conflicts with --rule. It is useful to scan with a subset of rules from a large
+  /// set of rule definitions within a project.
+  #[clap(short = 'F', long, conflicts_with = "rule", value_name = "RULE_ID")]
```

Comment:
Hi @obreitwi, would you like to remove the short option here? I'm quite unsure `-f` should be reserved for `--format` or `--filter`. 

My thought is that new option should not have short option by default and add short form later.